google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/GetAccountHierarchy.java

```diff
@@ -260,6 +261,7 @@ private void runExample(GoogleAdsClient googleAdsClient, Long managerId, Long lo
           System.out.printf(
               "Unable to retrieve hierarchy for customer ID %d: %s%n",
               customerIdToSearchFrom, gae.getGoogleAdsFailure().getErrors(0).getMessage());
+          System.exit(1);
```

Comment:
For this one, @devchas specifically set it up to return `null`. @devchas , we want to preserve that behavior and _not_ change this line, correct?